sections/05_treat.md

```diff
@@ -208,54 +208,52 @@ which small molecules (also referred to as chemical compounds or ligands)
 effectively and specifically affect the activity of a target, such as a kinase,
 protein-protein interaction, or broader cellular phenotype.  `TODO: clarify
 desired outputs, what will be done with the top-ranked hits, one vs multiple
-hits, abandoning lead compounds` This screening process can serve as the first
-step in the long drug discovery pipeline, where novel chemicals are pursued for
-their ability to inhibit or enhance disease-relevant biological mechanisms.  The
-appeal of machine learning in this domain is the need to improve the efficiency
-of the initial high-throughput screens such that sufficient candidate active
-compounds can be identified without exhaustively screening libraries of hundreds
-of thousands or millions of chemicals.  `TODO: is the sufficient number target
-dependent?` This task has been treated variously as a classification,
-regression, or
-ranking problem. In reality, it does not fit neatly into any of those
-categories.  An ideal algorithm will rank a sufficient number of active
-compounds before the inactives, but the rankings of actives relative to other
-actives and inactives are less important
+hits, abandoning lead compounds` This screening process can serve as one of the
+first steps in the long drug discovery pipeline, where novel chemicals are
+pursued for their ability to inhibit or enhance disease-relevant biological
+mechanisms.  The appeal of machine learning in this domain is the need to
+improve the efficiency of the initial high-throughput screens such that
+sufficient candidate active compounds can be identified without exhaustively
+screening libraries of hundreds of thousands or millions of chemicals.  `TODO:
+is the sufficient number target dependent?` This task has been treated variously
+as a classification, regression, or ranking problem. In reality, it does not fit
+neatly into any of those categories.  An ideal algorithm will rank a sufficient
```

Comment:
I'm a little uneasy about the use of "active" and "inactive" here (would have gone for "potential candidates" or "potentially useful") but will bow to the experts.